Example/Emission/AppDelegate.m

```diff
@@ -131,16 +133,43 @@ - (void)dismissModalViewController;
   [navigationController.visibleViewController.navigationController dismissViewControllerAnimated:YES completion:nil];
 }
 
+static NSArray *sharedRoutingMap;
+
+- (NSArray *)routingMap
+{
+    if (!sharedRoutingMap) {
+      sharedRoutingMap = @[
+        @{
+          @"name" : @"Storybook",
+          @"router" : ^() {
+            return [[ARStorybookComponentViewController alloc] init];
+          }
+        },
+        @{
+           @"name" : @"Home",
+           @"router" : ^() {
+             return [[ARHomeComponentViewController alloc] init];
+           }
+        },
+        @{
+          @"name" : @"Artist",
+          @"router" : ^() {
+            return [[ARArtistComponentViewController alloc] initWithArtistID:ARTIST];
+          }
+        },
+      ];
+    }
```

Comment:
the if/else wasn't going to scale to three, figured this was the simplest, and allows for some dynamism